code/game/objects/structures/signs/signs_departments.dm

```diff
@@ -5,18 +5,32 @@
 
 ///////MEDBAY
 
-/obj/structure/sign/departments/medbay
+/obj/structure/sign/departments/med
```

Comment:
having "med", "med_alt", and "medbay" seems extremely confusing

Reply:
what do you sugget?

Reply:
Ugh, I see. "Med" just says med, and "Medbay" says medbay. Probably nothing better and just looks dumb.

Reply:
maybe medbay_med and medbay_med_alt?